lib/Client.php

```diff
@@ -405,7 +405,10 @@ protected function createCurlSettingsArray(RequestInterface $request): array
                 $nHeaders[] = $key.': '.$value;
             }
         }
-        $settings[CURLOPT_HTTPHEADER] = $nHeaders;
+
+        if (!empty($nHeaders)) {
```

Comment:
The phpstan analysis does not allow using `empty()` any more - it is a "dangerous" PHP function because so many things are "empty" (false, zero etc.

We known that `$nHeaders` is an array, so you can check for `count($nHeaders) < 0` or some way to detect that it is not the empty array `[]`

Do `composer phpstan` locally to see what it complains abouut.